packages/mui-joy/src/styles/extendTheme.ts

```diff
@@ -360,7 +361,7 @@ export default function extendTheme(themeInput?: ThemeInput): Theme {
       xl3: 900,
     },
     focus: {
-      selector: '&.Mui-focusVisible, &:focus-visible',
+      selector: `&.${generateUtilityClass('', 'focusVisible')}, &:focus-visible`,
```

Comment:
```suggestion
      selector: `&.${generateUtilityClass('Joy', 'focusVisible')}, &:focus-visible`,
```
Shouldn't it be joy?

Reply:
It does not matter here. The first argument usually is a component name, eg. `MuiButton` or `JoyButton` but the `focusVisible` is a state class name that will ignore the component name anyway.

https://github.com/siriwatknp/material-ui/blob/9404b2d4c52819789cd082751d2bc073393aa483/packages/mui-joy/src/className/index.ts#L5

I will add a comment.

Reply:
Ah, I confused it with the prefix that we are adding with this PR. Makes sense 👍